html/modules/custom/ghi_blocks/src/Traits/GlobalMapTrait.php

```diff
@@ -76,4 +76,21 @@ public static function getGlobalMapSettings() {
     return \Drupal::config('ghi_blocks.map_settings')->get();
   }
 
+  /**
+   * Get plan attachment Map disclaimer.
+   *
+   * @var string $langcode
+   *   The langue code of the plan.
+   *
+   * @return string
+   *   Return the translated disclaimer.
+   */
+  public function getPlanMapDisclaimer(string $langcode): string {
```

Comment:
If you make the `$langcode` argument optional, you can default it to 'en' in this function, instead of doing that when you call it, so
```
$langcode = $this->getCurrentPlanObject()->getPlanLanguage() ?? 'en';
$this->getPlanMapDisclaimer($langcode)
```
could become
```
$langcode = $this->getCurrentPlanObject()->getPlanLanguage();
$this->getPlanMapDisclaimer($langcode)
```
if 
```
public function getPlanMapDisclaimer(?string $langcode = NULL): string {
```